### PR TITLE
Fix loading paths

### PR DESCRIPTION
**Pull request by kmcd**
This _should_ fix the path loading ...

### CODE REVIEW

File: lib/flex_commerce_api/api_base.rb [hunk 1]
```diff
@@ -13,18 +13,21 @@
...
 require "flex_commerce_api/json_api_client_extension/flexible_connection"
-require "flex_commerce_api/json_api_client_extension/resource"
+require "flex_commerce_api/json_api_client_extension/parsers/parser"
+
 module FlexCommerceApi
   #
   # Base class for all flex commerce models
   #
-  class ApiBase < ::FlexCommerceApi::JsonApiClientExtension::Resource
+  class ApiBase < JsonApiClient::Resource
```
Comment (garytaylor) [inline lib/flex_commerce_api/api_base.rb]: Why do these changes here ? The idea was keep the base clasd all about our api and less about which client it uses. 
Changing to another client will get harder doing it this way.

File: lib/flex_commerce_api/json_api_client_extension/included_data.rb [hunk 2]
```diff
@@ -1,12 +1,26 @@
-module FlexCommerce
+require 'json_api_client/included_data'
+
+module FlexCommerceApi
   module JsonApiClientExtension
     class IncludedData < ::JsonApiClient::IncludedData
+      def initialize(result_set, *args)
+        super(result_set, *args)
+        @result_set = result_set
```
Comment (garytaylor) [inline lib/flex_commerce_api/json_api_client_extension/included_data.rb]: This is obviously a workaround as you are going outside of the included datas responsibility of the included data class. As long as youre ok with this ?

File: lib/flex_commerce_api/api_base.rb (hunk 1 above)
Comment (kmcd) [inline lib/flex_commerce_api/api_base.rb]: Tried using `:FlexCommerceApi::JsonApiClientExtension::Resource` but ran into all sorts of loading issues.

This was the only class that worked :(

File: lib/flex_commerce_api/json_api_client_extension/included_data.rb (hunk 2 above)
Comment (kmcd) [inline lib/flex_commerce_api/json_api_client_extension/included_data.rb]: Yeah, thought about creating a wrapper but didn't work out. You'd need to do a lot of re-writing to get true separation of concerns ...

**Comment by alexander-e1off** [comment]
Please update ruby gem version

**Pull request merged**